src/ngraph/runtime/cpu/cpu_builder.cpp

```diff
@@ -470,6 +470,18 @@ namespace ngraph
                 BUILD_UNARY_ELEMWISE_CF_FUNCTOR(runtime::cpu::kernel::sqrt);
             }
 
+            template <>
+            NodeExecutorTy Builder::BUILDER_CF_DECL(ngraph::op::Floor)
```

Comment:
No unit-tests ?

Reply:
@nmostafa I didn't realize until you nudged me on this that you guys _do_ have tests for these functors. I thought they were just going untested. I'll add some tests for the new functors I have been adding. Please switch to `changes requested`, or I can follow on in a later PR.

Reply:
Have added some tests.